libs/blocks/global-navigation/global-navigation.js

```diff
@@ -813,6 +813,11 @@ class Gnav {
     analyticsValue: 'Logo',
   });
 
+  decorateClientSearch = () => {
+    this.elements.customSearch = toFragment`<div class="feds-client-search"></div>`;
+    return this.elements.customSearch;
```

Comment:
It doesn't look like you're using `this.elements.customSearch` anywhere, so why not just
```suggestion
    return toFragment`<div class="feds-client-search"></div>`;
```

Alternatively, you could have the `searchEnabled` check here and return the element or `''` and then just call the method directly in `decorateMainNav`.